Umbraco Commerce How to Guide on Creating and Order via Codea (#7017)
umbraco/UmbracoDocs: 15/umbraco-commerce/how-to-guides/create-order-via-code.md

---
description: Learn how to create an order via code in Umbraco Commerce.
---

# Create an Order via Code

> It is sometimes necessary, such as when importing orders from another system, to create a fully finalized order via code. 

Reviewer: ```suggestion
In some cases, such as when importing orders from another system, it may be necessary to create a fully finalized order via code. 
```

> An example of the code to do this is shown below. Here the `api` variable is an instance of the `IUmbracoCommerceApi` interface, which is an injectable service you access via [dependency injection](../key-concepts/dependency-injection.md).

Reviewer: ```suggestion
The example below demonstrates how to do this. The `api` variable is an instance of the `IUmbracoCommerceApi` interface, which is an injectable service accessed via [dependency injection](../key-concepts/dependency-injection.md).
```
